src/serialization.c

```diff
@@ -115,6 +122,40 @@ static size_t calcSerializedSize(RaftRedisCommand *cmd)
     return sz;
 }
 
+static int decodeInteger(const char *ptr, size_t sz, char expect_prefix, size_t *val)
```

Comment:
It's not clear why we need this function and how exactly it behaves, I suggest to document its interface (same applies for other functions in this file, maybe it's time to LIBTYFI).